Makefile

```diff
@@ -2,7 +2,7 @@ SHELL=/bin/bash
 # Makefile for OpenHands project
 
 # Variables
-DOCKER_IMAGE = ghcr.io/openhands/sandbox:main
+DOCKER_IMAGE = ghcr.io/all-hands-ai/sandbox:main
```

Comment:
```suggestion
DOCKER_IMAGE = ghcr.io/all-hands-ai/sandbox:main
```

I think we can remove all occurrence of sandbox image now, since they are no longer used!

Reply:
Will remove that line.
But I'm not sure what the scope is with removing sandbox image everywhere, would rather see you do that in a separate PR, tbh.